applications/machine_learning/configuration/nrf52840dk_nrf52840/app.overlay

```diff
@@ -1,4 +1,8 @@
 / {
+	chosen {
+		nordic,ml_app_ei_data_forwarder_uart = &uart0;
```

Comment:
We tipically use `ncs,` prefix for NCS choices, `nordic` is a vendor prefix tipically used for compatible strings. Also please don't use undercores but `-`

Reply:
Updated with fix from @de-nordic